src/main/java/uk/gov/justice/digital/hmpps/keyworker/dto/AllocationHistoryDto.java

```diff
@@ -3,6 +3,7 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import io.swagger.annotations.ApiModelProperty;
```

Comment:
Is this still used? This comment applies to most of these files

Reply:
no i will remove, thought i had deleted, perhaps not.  Was expecting the linter to remove, running reformat / lint again

Reply:
hmm.. you are right - the linter should have picked it up. This is one of the first APIs so i think it hasn't got the linter set up properly. That would be worth investigating some time.